DARPA-ASKEM/terarium
# feat: dataset enrichment and new layout

# Description

* new enrichment button/modal
* moved `tera-dataset-description` contents into `tera-dataset` since there is no need to separate them
* will update the column information section's deisgn in another PR since this PR is pretty huge

![image](https://github.com/user-attachments/assets/f28354f4-014e-46ec-93de-451c145311be)

* enriching works, the checkbox for overwriting or not will be handled in a second pass
![image](https://github.com/user-attachments/assets/84d0f20d-ca3a-4a6a-9cfa-4a8d128a480d)

### Testing
- Click the header button "Enrich metadata with AI assistant"
- Enrich the dataset using No document or a document you have in your project
- See if your Column Information is updated
